tests/specs/functional/storageRoot-spec.k

```diff
@@ -21,7 +21,7 @@ module STORAGEROOT-SPEC
          </k>
 
     rule <k> runMerkle( Keccak256( #rlpEncodeMerkleTree( #storageRoot(
-              #hashedLocation( "Solidity", keccakIntList(0), .IntList )                         |-> 1234
+              #hashedLocation( "Vyper", 0, 0 )                                                  |-> 1234
```

Comment:
This is arguably hackier, since we are mixing compiler versions now. Does `"Solidity", 0, 0` not work?